pkg/apis/meta.go

```diff
@@ -0,0 +1,303 @@
+package apis
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// Annotation keys
+
+const AnnoKeyCreatedBy = "grafana.app/createdBy"
+const AnnoKeyUpdatedTimestamp = "grafana.app/updatedTimestamp"
+const AnnoKeyUpdatedBy = "grafana.app/updatedBy"
+const AnnoKeyFolder = "grafana.app/folder"
+const AnnoKeySlug = "grafana.app/slug"
+const AnnoKeyTitle = "grafana.app/title"
```

Comment:
```suggestion
```
Can we remove the annotation flavor of this?  feels too confusing to have multiple ways

Reply:
sure, easy enough to rip it out.